src/uadk.h

```diff
@@ -17,6 +17,9 @@
 #include <openssl/engine.h>
 #include <uadk/wd.h>
 
+#define OPENSSL_SUCCESS 1
+#define OPENSSL_FAIL    0
+
```

Comment:
do not change return value in this pr, still need discussion.
if change, we need change all in one patch

https://www.openssl.org/policies/codingstyle.txt
Functions can return values of many different kinds, and one of the
most common is a value indicating whether the function succeeded or
failed. Usually this is:

    1: success
    0: failure

Sometimes an additional value is used:

    -1: something bad (e.g., internal error or memory allocation failure)

Other APIs use the following pattern:

    >= 1: success, with value returning additional information
    <= 0: failure with return value indicating why things failed